### PR TITLE
feat: enable hooks and video examples 

### PR DESCRIPTION
**Pull request by abretonc7s**
Adjust overall project docs
- enable pre commit hooks
- force conventional commits

### CODE REVIEW

File: packages/devreactnative/ios/devreactnative.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -486,6 +486,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_TEAM = 48XVW22RCG;
```
Comment (andreahaku) [inline packages/devreactnative/ios/devreactnative.xcodeproj/project.pbxproj:489]: take out this please

File: packages/devreactnative/ios/devreactnative.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -513,6 +514,7 @@
...
```
Comment (andreahaku) [inline packages/devreactnative/ios/devreactnative.xcodeproj/project.pbxproj:517]: same

**Review by andreahaku — COMMENTED** [review]
some changes required

**Review by andreahaku — DISMISSED** [review]
LGTM

**Review by omridan159 — APPROVED** [review]

**Pull request merged**